bundle-workflow/dockerfiles/ci-runner.centos8.dockerfile

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# This is a docker image specifically for standardize the ci/cd environment
+# for both developers and ci/cd tools in OpenSearch / OpenSearch-Dashboards
+# Please read the README.md file for all the information before using this dockerfile
+
+
+FROM centos:8
+
+# Add AdoptOpenJDK Repo
+RUN echo -e "[AdoptOpenJDK]\nname=AdoptOpenJDK\nbaseurl=http://adoptopenjdk.jfrog.io/adoptopenjdk/rpm/centos/7/\$basearch\nenabled=1\ngpgcheck=1\ngpgkey=https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public" > /etc/yum.repos.d/adoptopenjdk.repo
+
+# Add normal dependencies
+RUN yum clean all && \
+    yum update -y && \
+    yum install -y adoptopenjdk-14-hotspot which curl git tar net-tools procps-ng cmake python3 python3-pip && \
+    ln -sfn `which pip3` /usr/bin/pip && pip3 install pipenv && pipenv --version 
+
+# Add Dashboards dependencies
```

Comment:
What are your thoughts on having multiple images vs a single image? Could we setup a base image for our CI systems and layer on top OpenSearch, dashboard, or other necessary components.

Is there much of  size/boot up time different with these dashboard/notebook specific dependences?

Reply:
Boot up is not a problem, and I am trying to reduce the number of images by adding all the dependencies.
There is no reason to have a base image again since we already based on minimal AL2/CentOS8.